src/spryker/harness.yml

```
@@ -89,7 +89,7 @@ attributes:
       php-fpm: "= 'my127/spryker:' ~ @('php.version') ~ '-fpm-' ~ (@('php.version') >= 7.4 ? 'buster' : 'stretch')"
   elasticsearch:
     image: elasticsearch
-    tag: '7.16.2'
+    tag: "= @('spryker.demoshop-version') <= '202009.0'? '6.8.22' : '7.16.2'"
```

Since demoshop-version is hard coded, can we either:

1) update UPGRADE.md to mention projects should put their demoshop version in workspace.yml
2) (1) and copy the demoshop-version into workspace.yml on `ws create`
3) use auto-detection if demoshop-version not supplied, with it not being supplied by default (harness can still reference another variable for ws create)

---

also upgrading ES to the latest version
```suggestion
    tag: "= @('spryker.demoshop-version') <= '202009.0'? '6.8.23' : '7.16.3'"
```